egs2/libritts/asr1/run.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+train_set="train-960"
+valid_set="dev"
+test_sets="test-clean test-other dev-clean dev-other"
+
+asr_config=conf/train_asr.yaml
+inference_config=conf/decode_asr.yaml
+
+./asr.sh \
+    --lang en \
+    --ngpu 2 \
+    --nbpe 100 \
```

Comment:
Is 100 bpe size an empirical good number?

Reply:
Yes, it provided the best average PER in preliminary experiments:

<table>
  <tr>
    <td rowspan="2"> </td>
    <td colspan="2" align="center">LS</td>
   <td colspan="2" align="center">VCTK</td>
   <td rowspan="2">Avg.</td>
  </tr>
  <tr>
    <td>dev</td>
    <td>test</td>
    <td>dev</td>
    <td>test</td>
  </tr>
 <tr>
    <td>Char</td>
    <td>7.5</td>
    <td>7.4</td>
    <td>7.9</td>
    <td>11.7</td>
   <td>8.63</td>
  </tr>
 <tr>
    <td>BPE 100</td>
    <td>7.4</td>
    <td>7.2</td>
    <td><b>6.6</b></td>
    <td><b>10.7</b></td>
   <td><b>7.98</b></td>
  </tr>
 <tr>
    <td>BPE 200</td>
    <td><b>7.0</b></td>
    <td><b>6.9</b></td>
    <td>7.2</td>
    <td>11.1</td>
   <td>8.05</td>
  </tr>

</table>